plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterResponse.py

```diff
@@ -16,13 +16,14 @@ class CloudClusterResponse(BaseCloudModel):
     #  \param status: The status of the cluster authentication (active or inactive).
     #  \param host_version: The firmware version of the cluster host. This is where the Stardust client is running on.
     def __init__(self, cluster_id: str, host_guid: str, host_name: str, is_online: bool, status: str,
-                 host_version: Optional[str] = None, **kwargs) -> None:
+                 host_internal_ip: str, host_version: Optional[str] = None, **kwargs) -> None:
```

Comment:
This should also be an Optional[str] as the field is not available in clusters running 5.2 (Beta) firmware.